201/ReadMe.md

```diff
@@ -28,9 +28,9 @@ Arc-Enabled KeyVault is an Arc extension that can be deployed to Arc-enabled clu
   
     Please take note of the Client ID (a.k.a., App ID), tenantID, service principal object ID and Client Secret generated in this step.
 
-2. Use the client ID and client secret as plain text from the first step to create a Kubernetes secret on the connected cluster:
+2. Use the client ID and client secret as plain text from the first step to create a Kubernetes secret on the connected cluster.
```

Comment:
Consider emphasizing client secret value rather than client secret id in line 31? I think saying "client secret value" in line 33 is still not attractive enough.

Reply:
Updated.